version.pri

```diff
@@ -24,3 +24,12 @@ DEFINES += QBT_VERSION_BUILD=$${VER_BUILD}
 
 DEFINES += QBT_VERSION=\\\"v$${PROJECT_VERSION}\\\"
 DEFINES += QBT_VERSION_2=\\\"$${PROJECT_VERSION}\\\"
+
+# only set git hash macro in release builds
+CONFIG(release, debug|release) {
+    DEFINES += QBT_GIT_HASH=\\\"$$system(git rev-parse --short=10 HEAD $$system_quote(--git-dir=$$PWD/.git))\\\"
```

Comment:
What if there is no git command?

Reply:
I've added some checks for this. CMake automatically detects this.